src/components/HomePage/Forms/EditMovieForm.tsx

```diff
@@ -1,132 +1,33 @@
 import React, { FC } from 'react';
-import {
-  Button, Form, FormGroup, Input, Label,
-} from 'reactstrap';
+import { Form } from 'reactstrap';
 import { MovieInfo } from '../../../types';
-import './Form.css';
+import { ResetButton, SubmitButton } from '../../shared';
+import './Form.scss';
+import { InputWithLabel } from './InputWithLabel';
+import { SelectInputGenre } from './SelectInputGenre';
 
-export const EditMovieForm: FC<{movieInfo: MovieInfo}>  = (props) => {
-  const movieGenres = ['Action', 'Adventure', 'Animation',
-    'Comedy', 'Drama', 'Family',
-    'Fantasy', 'Horror', 'Music',
-    'Mystery', 'Romance', 'Science Fiction',
-    'Thriller'];
-    // todo: this edit form will be updated with relevant data later (in next tasks)
-  
+export const EditMovieForm: FC<{movieInfo: MovieInfo}> = (props) => {
```

Comment:
You can use props destructuring right here `({ movieInfo })` instead of `(props)`

Reply:
Thank you. Updated in commit d0df8953a6498f884fc8f239cf86721b7a4c4d9a